### PR TITLE
Add documentation for Azure workload identity 

### PR DESCRIPTION
**Pull request by Reshrahim**
Thank you for helping make the Radius documentation better!

**Please follow this checklist before submitting:**

- [ ] [Read the contribution guide](https://docs.radapp.io/community/contributing/docs/)
- [ ] Commands include options for Linux, MacOS, and Windows within codetabs
- [ ] New file and folder names are globally unique
- [ ] Page references use shortcodes instead of markdown or URL links
- [ ] Images use HTML style and have alternative text
- [ ] Places where multiple code/command options are given have codetabs

In addition, please fill out the following to help reviewers understand this pull request:

## Description

This PR covers the Docs for Azure workload identity

### CODE REVIEW

File: docs/content/guides/operations/providers/azure-provider/howto-azure-provider-wi/index.md [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+
+## Prerequisites
+
+- [Azure subscription](https://azure.com)
+- [az CLI](https://aka.ms/azcli)
+- [rad CLI]({{< ref "installation#step-1-install-the-rad-cli" >}})
+- [Setup a supported Kubernetes cluster]({{< ref "/guides/operations/kubernetes/overview#supported-clusters" >}})
+- [Azure AD Workload Identity](https://azure.github.io/azure-workload-identity/docs/installation.html) installed in your cluster, including the [Mutating Admission Webhook](https://azure.github.io/azure-workload-identity/docs/installation/mutating-admission-webhook.html)
+-	Create an app registration at Microsoft Entra ID 
+-	Configure the federated credential for Radius components to deploy Azure resources following the script [here](insertscript) or manually configure the federated credential following the steps [here](https://learn.microsoft.com/en-us/entra/workload-id/workload-identity-federation-create-trust?pivots=identity-wif-apps-methods-azp#kubernetes)
```
Comment (Reshrahim) [inline docs/content/guides/operations/providers/azure-provider/howto-azure-provider-wi/index.md]: @willdavsmith  - can you help add the details of the service account and namespace here

File: docs/content/guides/operations/providers/azure-provider/howto-azure-provider-wi/index.md [hunk 2]
```diff
@@ -19,13 +19,94 @@ The Azure provider allows you to deploy and connect to Azure resources from a se
...
--	Configure the federated credential for Radius components to deploy Azure resources following the script [here](insertscript) or manually configure the federated credential following the steps [here](https://learn.microsoft.com/en-us/entra/workload-id/workload-identity-federation-create-trust?pivots=identity-wif-apps-methods-azp#kubernetes) 
-   | Cluster Issuer URL | Service account name | Namespace | name |
-   |---------------------|----------------------|-----------|------|
-   |  |  |  |  |
-
+- [Entra ID Application Registration with federated credentials]()
+
+## Setup
+
+### install-radius-azwi.sh
```
Comment (Reshrahim) [inline docs/content/guides/operations/providers/azure-provider/howto-azure-provider-wi/index.md]: @willdavsmith - Can you add some details on what the script does? If they do not wish use the script, what config they need to do manually

File: docs/content/guides/operations/providers/azure-provider/howto-azure-provider-wi/index.md [hunk 3]
```diff
@@ -0,0 +1,97 @@
...
+- [Azure subscription](https://azure.com)
+- [az CLI](https://aka.ms/azcli)
+- [rad CLI]({{< ref "installation#step-1-install-the-rad-cli" >}})
+- [Setup a supported Kubernetes cluster]({{< ref "/guides/operations/kubernetes/overview#supported-clusters" >}})
+  - You will need the cluster's OIDC Issuer URL. [AKS Example](https://azure.github.io/azure-workload-identity/docs/installation/managed-clusters.html#azure-kubernetes-service-aks)
+- [Azure AD Workload Identity](https://azure.github.io/azure-workload-identity/docs/installation.html) installed in your cluster, including the [Mutating Admission Webhook](https://azure.github.io/azure-workload-identity/docs/installation/mutating-admission-webhook.html)
+
+## Setup the Azure Workload Identity for Radius
+
+To authorize Radius with Azure using Azure workload identity, you should set up an Entra ID Application with access to your resource group and 3 federated credentials (one for each of the Radius services). The 3 federated credentials should be created with the Kubernetes ServiceAccounts for each of the Radius services (applications-rp, bicep-de, and ucp) and the OIDC Issuer for your Kubernetes cluster.
```
Comment (Reshrahim) [inline docs/content/guides/operations/providers/azure-provider/howto-azure-provider-wi/index.md]:
```suggestion
To authorize Radius with Azure using Azure workload identity, you should set up an Entra ID Application with access to your resource group and 3 federated credentials (one for each of the Radius services). The 3 federated credentials should be created with the Kubernetes ServiceAccounts for each of the Radius services (applications-rp, bicep-de, and ucp) in the `radius-system` namespace and the OIDC Issuer for your Kubernetes cluster.
```

File: docs/content/guides/operations/providers/azure-provider/howto-azure-provider-wi/index.md [hunk 4]
```diff
@@ -0,0 +1,97 @@
...
+- [az CLI](https://aka.ms/azcli)
+- [rad CLI]({{< ref "installation#step-1-install-the-rad-cli" >}})
+- [Setup a supported Kubernetes cluster]({{< ref "/guides/operations/kubernetes/overview#supported-clusters" >}})
+  - You will need the cluster's OIDC Issuer URL. [AKS Example](https://azure.github.io/azure-workload-identity/docs/installation/managed-clusters.html#azure-kubernetes-service-aks)
+- [Azure AD Workload Identity](https://azure.github.io/azure-workload-identity/docs/installation.html) installed in your cluster, including the [Mutating Admission Webhook](https://azure.github.io/azure-workload-identity/docs/installation/mutating-admission-webhook.html)
+
+## Setup the Azure Workload Identity for Radius
+
+To authorize Radius with Azure using Azure workload identity, you should set up an Entra ID Application with access to your resource group and 3 federated credentials (one for each of the Radius services). The 3 federated credentials should be created with the Kubernetes ServiceAccounts for each of the Radius services (applications-rp, bicep-de, and ucp) in the `radius-system` namespace and the OIDC Issuer for your Kubernetes cluster.
```
Comment (Reshrahim) [inline docs/content/guides/operations/providers/azure-provider/howto-azure-provider-wi/index.md]:
```suggestion
To authorize Radius to connect to Azure using Azure workload identity, you should set up an Entra ID Application with access to your resource group and 3 federated credentials (one for each of the Radius services). The 3 federated credentials should be created with the Kubernetes ServiceAccounts for each of the Radius services (applications-rp, bicep-de, and ucp) in the `radius-system` namespace and the OIDC Issuer for your Kubernetes cluster.
```

File: docs/content/guides/author-apps/aws/howto-aws-resources/index.md [hunk 5]
```diff
@@ -49,7 +49,7 @@ Follow the prompts to install the [control plane services]({{< ref "/concepts/te
 - **Namespace** - When an application is deployed, this is the namespace where your containers and other Kubernetes resources will be run. By default, this will be in the `default` namespace.
 {{% alert title="💡 About namespaces" color="success" %}} When you initialize a Radius Kubernetes environment, Radius installs the control plane resources within    the `radius-system` namespace in your cluster, separate from your applications. The namespace specified in this step will be used for your application deployments.
 {{% /alert %}}
-- **Add AWS provider** - An [AWS cloud provider]({{< ref "/guides/operations/providers/howto-aws-provider" >}}) allows you to deploy and manage AWS resources as part of your application. Enter 'y' and follow the instructions. Provide a valid AWS region and the values obtained for IAM Access Key ID and IAM Secret Access Keys.
+- **Add AWS provider** - An [AWS cloud provider]({{< ref "/guides/operations/providers/aws-provider" >}}) allows you to deploy and manage AWS resources as part of your application. Follow the how-to guides to configure the AWS provider with the preferred identity.
```
Comment (willtsai) [inline docs/content/guides/author-apps/aws/howto-aws-resources/index.md]: recommend making `how-to guides` a hyperlink to the guide it is referring to.

File: docs/content/guides/operations/providers/azure-provider/howto-azure-provider-wi/index.md [hunk 6]
```diff
@@ -0,0 +1,97 @@
...
+
+Below is an example script that will create an Entra ID Application and set up the federated credentials necessary for Radius to authenticate with Azure using Azure workload identity.
+
+{{< rad file="snippets/install-radius-azwi.sh" embed=true >}}
+
+Now that the setup is complete, you can now install Radius with Azure workload identity enabled.
+
+## Interactive configuration
+
+1. Initialize a new environment with [`rad init --full`]({{< ref rad_init >}}):
```
Comment (willtsai) [inline docs/content/guides/operations/providers/azure-provider/howto-azure-provider-wi/index.md:37]: should the ref here be `reference/cli/rad_init` instead? Or do CLI command refs follow a different convention? Whatever it ends up being, please confirm that the link works in a local build.

File: docs/content/guides/operations/providers/azure-provider/howto-azure-provider-sp/index.md [hunk 7]
```diff
@@ -29,8 +29,10 @@ The Azure provider allows you to deploy and connect to Azure resources from a se
 
 1. Follow the prompts, specifying:
    - **Namespace** - The Kubernetes namespace where your application containers and networking resources will be deployed (different than the Radius control-plane namespace, `radius-system`)
-   - **Add an Azure provider** - Pick the subscription and resource group to deploy your Azure resources to
-      Run `az ad sp create-for-rbac` to create a Service Principal without a role assignment and obtain your `appId`, `displayName`, `password`, and `tenant` information.
+   - **Add an Azure provider** 
+      1. Pick the subscription and resource group to deploy your Azure resources to
+      2. Select the service principal option
```
Comment (willtsai) [inline docs/content/guides/operations/providers/azure-provider/howto-azure-provider-sp/index.md]:
```suggestion
      2. Select the "Service Principal" option
```

File: docs/content/guides/operations/providers/azure-provider/howto-azure-provider-wi/index.md (hunk 6 above)
Comment (Reshrahim) [reply docs/content/guides/operations/providers/azure-provider/howto-azure-provider-wi/index.md:37]: yeah the reference works. We reference with absolute paths for pages that have folders with same names. For eg : there is a k8s folder both under authoring and operations where we need absolute path.

File: docs/content/guides/author-apps/aws/howto-aws-resources/index.md (hunk 5 above)
Comment (Reshrahim) [reply docs/content/guides/author-apps/aws/howto-aws-resources/index.md]: added

**Review by willtsai — APPROVED** [review]
LGTM, but please get the spellcheck to pass before merging.

**Pull request merged**